quickstart-build/cloudbuild.yaml

```diff
@@ -15,7 +15,9 @@
 # [START cloudbuild_quickstart_build]
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: [ 'build', '-t', 'us-west2-docker.pkg.dev/$PROJECT_ID/quickstart-docker-repo/quickstart-image:tag1', '.' ]
+  script: |
+    docker build -t us-west2-docker.pkg.dev/$PROJECT_ID/quickstart-docker-repo/quickstart-image:tag1 .
+    automapSubstitutions: true
```

Comment:
Extra indentation here, `automapSubstitutions` is on the step level so it should be indented the same as `script`

Reply:
thanks @GiangN!